doc/classes/Window.xml

```diff
@@ -365,6 +365,12 @@
 				Centers a native window on the current screen and an embedded window on its embedder [Viewport].
 			</description>
 		</method>
+		<method name="move_to_mouse">
```

Comment:
This fails because it is not in alphabetical order.

Don't edit the XML structure manually, instead, run `--doctool` on the generated Godot executable to regenerate the documentation.